docs/chains/celestia-chain/celestia-node.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: /celestia-node
+title: Running a celestia RPC Node
+---
+
+# Running a celestia RPC Node
+
+## Requirements 📄
+
+### Recommended Specifications
+
+    CPU: 4+ cores
+    RAM: 8 GB+
+    Disk 1 TB
+    Connection Speed: 25+ mbps/sec
+
+
+##  Setup celestia consensus and DA node
+
+Follow celestia docs for instructions on how to setup and run the nodes:
+Data availabillity light node https://docs.celestia.org/nodes/light-node
+Consensus node https://docs.celestia.org/nodes/full-consensus-node
+
+Important note:
+When running the light node make sure to add the flag --rpc.skip-auth to let consumers use the node without authentication
```

Comment:
let the provider service, not consumers, they won't have direct access to the node, let's also add the authenticated path to the docs